src/main/resources/story/walmart/VisualTestEmptyCart.story

```diff
@@ -0,0 +1,11 @@
+Description: Story to test empty cart view;
+
+Scenario: Test page context view
+Given I am on a page with the URL 'https://www.walmart.com/cart'
```

Comment:
I stored ''https://www.walmart.com' to 'web-application.main-page-url'. Can I somehow address to 'https://www.walmart.com/cart' using relative path '/cart'?

Reply:
You can use this step "When I go to the relative URL '/cart'"